sdks/go/pkg/beam/core/runtime/exec/datasource.go

```diff
@@ -441,13 +457,15 @@ func (n *DataSource) Progress() ProgressReportSnapshot {
 	// The count is the number of "completely processed elements"
 	// which matches the index of the currently processing element.
 	c := n.index
+	// Retrieve the signal from the Data source.
```

Comment:
Similarly redundant comment, due to the variable naming.

Reply:
Done!